ops/test/api.tf

```diff
@@ -12,6 +12,7 @@ module "simple_report_api" {
   docker_image_uri = "DOCKER|simplereportacr.azurecr.io/api/simple-report-api-build:${var.acr_image_tag}"
   key_vault_id     = data.azurerm_key_vault.sr_global.id
   tenant_id        = data.azurerm_client_config.current.tenant_id
+  https_only       = true
```

Comment:
Is there not a place where this could be applied centrally to all environments? That seems weird.

Reply:
There is. Do we need to allow any environment to accept plaintext traffic?